core/udemy.py

```diff
@@ -77,6 +79,21 @@ def redeem(self, url: str) -> None:
             if element_text not in self.settings.languages:
                 print(f"Course language not wanted: {element_text}")
                 return
+        if self.settings.categories:
+            # If the wanted categories are specified, get all the categories of the course by
+            # scraping the breadcrumbs on the top
+
+            breadcrumbs: WebElement = self.driver.find_element_by_class_name(
+                "udlite-breadcrumb"
+            )
+            breadcrumbs = breadcrumbs.find_elements_by_class_name("udlite-heading-sm")
+            breadcrumbs = [bc.text for bc in breadcrumbs]  # Get only the text
+
+            for category in self.settings.categories:
+                if category in breadcrumbs:
+                    break
+            else:
+                return print(f"Skipping course as it does not have a wanted category")
```

Comment:
```suggestion
                print(f"Skipping course as it does not have a wanted category")
                return
```

Reply:
why not `return print()`? It is pretty much the same thing right?

Reply:
Ye it does the same thing but if you're not familiar with python you might expect the return to contain the text of print

Using `return` is explicit in saying it returns `None` as there is literally nothing being returned.

What you have does the same thing its just clearer if its split out and in line with the rest of the code.

Reply:
Okay sure.